### PR TITLE
CASSANDRA-19201: Refactor cqlshmain global constants

### PR DESCRIPTION
**Pull request by bschoening**
Thanks for sending a pull request! Here are some tips if you're new here:
 
 * Ensure you have added or run the [appropriate tests](https://cassandra.apache.org/_/development/testing.html) for your PR.
 * Be sure to keep the PR description updated to reflect all changes.
 * Write your PR title to summarize what this PR proposes.
 * If possible, provide a concise example to reproduce the issue for a faster review.
 * Read our [contributor guidelines](https://cassandra.apache.org/_/development/index.html)
 * If you're making a documentation change, see our [guide to documentation contribution](https://cassandra.apache.org/_/development/documentation.html)
 
Commit messages should follow the following format:

```
<One sentence description, usually Jira title or CHANGES.txt summary>

<Optional lengthier description (context on patch)>

patch by <Authors>; reviewed by <Reviewers> for CASSANDRA-#####

Co-authored-by: Name1 <email1>
Co-authored-by: Name2 <email2>

```

The [Cassandra Jira](https://issues.apache.org/jira/projects/CASSANDRA/issues/)

### CODE REVIEW

File: pylib/cqlshlib/cqlshmain.py [hunk 1]
```diff
@@ -2206,30 +2126,15 @@ def read_options(cmdlineargs, environment=os.environ):
...
-    cqlruleset = cql3handling.CqlRuleSet
-    cqlruleset.append_rules(cqlshhandling.cqlsh_extra_syntax_rules)
-    for rulename, termname, func in cqlshhandling.cqlsh_syntax_completers:
-        cqlruleset.completer_for(rulename, termname)(func)
-    cqlruleset.commands_end_with_newline.update(cqlshhandling.my_commands_ending_with_newline)
-
-
-def setup_docspath(path):
-    global CASSANDRA_CQL_HTML
-    CASSANDRA_CQL_HTML_FALLBACK = 'https://cassandra.apache.org/doc/latest/cassandra/cql/cql_singlefile.html'
```
Comment (smiklosovic) [inline pylib/cqlshlib/cqlshmain.py:2221]: @bschoening where did this go?

File: pylib/cqlshlib/cqlshmain.py [hunk 2]
```diff
@@ -85,83 +86,11 @@
...
-DEFAULT_SSL = False
-DEFAULT_CONNECT_TIMEOUT_SECONDS = 5
-DEFAULT_REQUEST_TIMEOUT_SECONDS = 10
-
-DEFAULT_FLOAT_PRECISION = 5
-DEFAULT_DOUBLE_PRECISION = 5
-DEFAULT_MAX_TRACE_WAIT = 10
-
 if readline is not None and readline.__doc__ is not None and 'libedit' in readline.__doc__:
     DEFAULT_COMPLETEKEY = '\t'
```
Comment (smiklosovic) [inline pylib/cqlshlib/cqlshmain.py:90]: @bschoening can't be this moved to `Shell` as well? ... whole `if`

File: pylib/cqlshlib/cqlshmain.py (hunk 1 above)
Comment (smiklosovic) [reply pylib/cqlshlib/cqlshmain.py:2221]: aha I see, into DEFAULT_CQLDOCS_URL

File: pylib/cqlshlib/cqlshmain.py (hunk 2 above)
Comment (bschoening) [reply pylib/cqlshlib/cqlshmain.py:90]: @smiklosovic I was planning to do that as part of [CASSANDRA-18914](https://issues.apache.org/jira/browse/CASSANDRA-18914).  Moving the logic and the globals got too mess, so this focuses mostly on refactoring the global constants.

**Pull request closed without merging**